lib/vm.js

```diff
@@ -54,4 +54,11 @@ exports.runInThisContext = function(code, options) {
   return script.runInThisContext(options);
 };
 
+exports.runInGlobalContext = function(code, options) {
+  var script = new Script(require('module').wrap(code), options);
+  return script.runInThisContext(options)(exports, require, module,
+                                                            __filename,
+                                                            __dirname);
```

Comment:
That's not 'run in global context', that's 'run in the context of the vm module'.